package.json

```diff
@@ -99,7 +99,7 @@
     "storybook": "storybook dev -p 6006",
     "build-storybook": "storybook build",
     "format": "prettier --cache --write './**/*.{ts,tsx,scss,css,json,md,mdx}'",
-    "lint": "eslint './**/*.{js,jsx,ts,tsx}' --max-warnings 0",
+    "lint": "tsc && eslint './**/*.{js,jsx,ts,tsx}' --max-warnings 0",
```

Comment:
@angry-dan I don't think `tsc` is required now as it's part of eslint typescript checks now, that's correct isn't it @3LivesLeft from when we originally removed `tsc`

Reply:
I had a build fail when I was working on this branch because of an invalid type which eslint didn't pick up. 

https://github.com/tpximpact/dx-component-quickstart/actions/runs/7398839976/job/20128949317

Reply:
Ah interesting in that case ignore me and keep `tsc` in for better coverage